renderdoccmd/CMakeLists.txt

```diff
@@ -245,11 +245,11 @@ if(ANDROID)
                 )
         endif()
     endif()
-    
+
     add_custom_command(OUTPUT ${APK_FILE} APPEND
                        COMMAND ${BUILD_TOOLS}/aapt package -f -m -S res -J src -M AndroidManifest.xml -I ${ANDROID_JAR}
                        COMMAND ${JAVA_BIN}/javac -d ./obj -source 1.7 -target 1.7 -bootclasspath ${RT_JAR} -classpath "${CLASS_PATH}" -sourcepath src src/org/renderdoc/renderdoccmd/*.java
-                       COMMAND ${BUILD_TOOLS}/dx${TOOL_SCRIPT_EXTENSION} --dex --output=bin/classes.dex ./obj
+                       COMMAND ${BUILD_TOOLS}/d8 --output ./bin/ ./obj/org/renderdoc/renderdoccmd/*/*.class
```

Comment:
You've removed the `${TOOL_SCRIPT_EXTENSION}` here but that's important to ensure this runs correctly on windows. `.bat` files aren't executed in all cases when invoked without an extension like this.

Reply:
I couldn't remember the third todo for keeping this a draft; that is indeed validating whether `d8` is an executable or remains a batch script on Windows.